app/main/auto-updater/index.js

```diff
@@ -13,10 +13,11 @@ import _ from 'lodash';
 import env from '../../env';
 
 const isDev = process.env.NODE_ENV === 'development';
+const runningLocally = isDev || process.env.RUNNING_LOCALLY;
   
 let checkNewUpdates = _.noop;
 
-if (!isDev) {
+if (!isDev && !runningLocally) {
```

Comment:
The script `npm start` runs the unpackaged Electron app, which is unsigned. If it's unsigned, trying to run `autoUpdate` will cause the app to break, because autoUpdate requires apps to be signed. 

Add this flag to skip auto update so we can still test unpackaged, unsigned builds.